htdocs/product/stock/replenish.php

```diff
@@ -43,6 +43,9 @@
 }
 $result=restrictedArea($user,'produit|service');
 
+// Initialize technical object to manage hooks of page. Note that conf->hooks_modules contains array of hook context
+$hookmanager->initHooks(array('stockreplenishlist', 'globalcard'));
```

Comment:
Can you push PR on develop
Also remove "globalcard" since this hook context is dedicated for a tab of a record, not for a list.

Reply:
No problem.
"globalcard" is removed and PR will be moved